### PR TITLE
feat: ga4 support

### PR DESCRIPTION
**Pull request by gasoved**
#29

### CODE REVIEW

**Review by julianlam — CHANGES_REQUESTED** [review]

File: library.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	await plugin.loadSettings();
+};
+
+plugin.filterConfigGet = async function (config) {
+	config[PLUGIN_NAME] = plugin.settings;
+	return config;
+};
+
+plugin.loadSettings = async () => {
+	const settings = await meta.settings.get(PLUGIN_NAME);
```
[the remark this answers is not in the document]
Comment (julianlam) [reply library.js:29]: ... also this means you don't need to listen for `action:settings.set` :smile:

**Comment by julianlam** [comment]
LGTM otherwise, the settings fix you don't have to update if you don't want to, it's mostly a note for the future. But let me know if you intend to update it here.

**Comment by gasoved** [comment]
"Settings fix" is ready, testing it locally right now. :+1:

File: library.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (gasoved) [reply library.js]: Done.

**Pull request merged**